loader/loader_test.go

```diff
@@ -281,13 +301,15 @@ var _ = Describe("Release Adapter", Ordered, func() {
 
 	Context("When calling GetProcessingResources", func() {
 		It("returns all the relevant resources", func() {
+			os.Setenv("ENTERPRISE_CONTRACT_CONFIG_MAP", "default/ec-defaults")
```

Comment:
should handle if the assignment failed for any reason ? like in  [ main.go Line137](https://github.com/redhat-appstudio/release-service/pull/205/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R137)